index.html

```diff
@@ -48,164 +54,206 @@ <h1 class="hey-text">Hey there. I’m Mariane <br>
             <li class="ruby-2">Ruby </li>
             <li class="html-2">Html</li>
             <li class="css-2">CSS</li>
-       </ul>
-       </div>
-
-  <!-- section-2-->
-  <div class="sections ">
-     <div class="language-icon"><img src="images/about-me/icon-frameworks.svg" alt="Rectangle icones"></div>
-     <h4>Frameworks</h4>
-     <ul class="frameworks">
-       <li class="bootstrap">Bootstrap</li>
-       <li class="ruby-3">Ruby on rails</li>
-       <li class="rspec">RSpec</li>
-       <li class="capybara">Capybara</li>
-       <li class="selenium">Selenium</li>
-    </ul>
-  </div>
-  <!--section-3-->
-  <div class="sections ">
-   <div class="language-icon"><img src="images/about-me/icon-skills.svg" alt="lightning icones"></div>
-   <h4>Skills</h4>
-   <ul class="frameworks">
-     <li class="codekit">Codekit</li>
-     <li class="github-2">Github</li>
-     <li class="codepen">Codepen</li>
-     <li class="gitlab">Gitlab</li>
-     <li class="terminal">Terminal</li>
-   </ul>
-   </div>
-
+          </ul>
+        </div>
+        
+        <!-- section-2-->
+        <div class="sections ">
+          <div class="language-icon"><img src="images/about-me/icon-frameworks.svg" alt="Rectangle icones"></div>
+          <h4>Frameworks</h4>
+          <ul class="frameworks">
+            <li class="bootstrap">Bootstrap</li>
+            <li class="ruby-3">Ruby on rails</li>
+            <li class="rspec">RSpec</li>
+            <li class="capybara">Capybara</li>
+            <li class="selenium">Selenium</li>
+          </ul>
+        </div>
+        <!--section-3-->
+        <div class="sections ">
+          <div class="language-icon"><img src="images/about-me/icon-skills.svg" alt="lightning icones"></div>
+          <h4>Skills</h4>
+          <ul class="frameworks">
+            <li class="codekit">Codekit</li>
+            <li class="github-2">Github</li>
+            <li class="codepen">Codepen</li>
+            <li class="gitlab">Gitlab</li>
+            <li class="terminal">Terminal</li>
+          </ul>
+        </div>
+      </div>
   </div>
   </section>
+    <!--Form section-->
+    <section class="form">
+      <div class="info">
+
+       <div class="form-text"><h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3></div>
+       <form action="https://formspree.io/f/xeqwozwl" method="post" >
+       
+         <input id="name" type="text" class="input"  name="user_name" placeholder="Full name" required maxlength="30">
+
+         <input id="email" type="email" class="input" name="user_email" placeholder="Email address" required>
+
+         <textarea id="message" class="msg" name="user_message" placeholder="Enter text here" required maxlength="500"></textarea>
+
+         <button type="submit" class="form-bttn" id="btnid">Get In Touch</button>
+         
+         
+       </form>
+       <footer>
+      </div>
+      <ul class="icons"> 
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give the right style to the `text` on the left and I think you have not created the long `Horizontal` line in the footer, please follow my suggestions above 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/221367194-864552a0-d8bf-43f2-b2ca-97ffcc37fa1b.png)